packages/alea-frontend/pages/api/job-portal/create-recruiter-profile.ts

```diff
@@ -0,0 +1,27 @@
+import { NextApiRequest, NextApiResponse } from 'next';
+import { checkIfPostOrSetError, executeAndEndSet500OnError } from '../comment-utils';
+import { getUserIdIfAuthorizedOrSetError } from '../access-control/resource-utils';
+import { Action, CURRENT_TERM, ResourceName } from '@stex-react/utils';
+
+export default async function handler(req: NextApiRequest, res: NextApiResponse) {
+  if (!checkIfPostOrSetError(req, res)) return;
+  const userId = await getUserIdIfAuthorizedOrSetError(
+    req,
+    res,
+    ResourceName.JOB_PORTAL,
+    Action.CREATE_JOB_POST,
+    { instanceId: CURRENT_TERM }
+  );
+  if (!userId) return;
+  const { name, email, position } = req.body;
+
```

Comment:
check if any of the inputs are empty and return 422. `email` should contain `@`

Add such checks for inputs in all your APIs